CMakeLists.txt

```diff
@@ -313,6 +327,9 @@ add_subdirectory(mgmt/config)
 add_subdirectory(mgmt/rpc)
 add_subdirectory(src/traffic_server)
 add_subdirectory(src/traffic_ctl)
+if(TS_USE_QUIC)
+    add_subdirectory(src/traffic_quic)
```

Comment:
Thanks for taking care of this, but actually we no longer maintain it. https://github.com/apache/trafficserver/pull/9726

Reply:
Thanks, I'll follow suite and obliterate it from the CMake build as well.